NUnitLite-1.0.0/src/framework/Internal/Results/TestResult.cs

```diff
@@ -412,20 +413,27 @@ public void RecordException(Exception ex)
             if (ex is NUnitException)
                 ex = ex.InnerException;
 
-            if (ex is System.Threading.ThreadAbortException)
+            if (ex is System.Threading.ThreadAbortException) {
```

Comment:
Please undo the unnecessary changes

Reply:
I have been repeatedly told that it is against our style to add braces around an `else` body without also adding it to the `if` body. 

I can undo this change, but I ask that we pick a consistent rule and stick to it.